tests/other/test_smtlibv2.py

```diff
@@ -26,82 +27,362 @@
+        check(
+            BoolUnsignedGreaterOrEqualThan, operanda=bvx, operandb=bvy, pickle_size=149, sizeof=48
+        )
+        check(BoolGreaterOrEqualThan, operanda=bvx, operandb=bvy, pickle_size=141, sizeof=48)
+        check(BoolUnsignedLessThan, operanda=bvx, operandb=bvy, pickle_size=139, sizeof=48)
+        check(BoolLessThan, operanda=bvx, operandb=bvy, pickle_size=131, sizeof=48)
+        check(BoolUnsignedLessOrEqualThan, operanda=bvx, operandb=bvy, pickle_size=146, sizeof=48)
+        check(BoolLessOrEqualThan, operanda=bvx, operandb=bvy, pickle_size=138, sizeof=48)
+
+        check(BitvecOr, operanda=bvx, operandb=bvy, pickle_size=129, sizeof=56)
+        check(BitvecXor, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecAnd, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecNot, operanda=bvx, pickle_size=112, sizeof=56)
+        check(BitvecNeg, operanda=bvx, pickle_size=112, sizeof=56)
+        check(BitvecAdd, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecMul, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecSub, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecDiv, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecMod, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecUnsignedDiv, operanda=bvx, operandb=bvy, pickle_size=138, sizeof=56)
+        check(BitvecRem, operanda=bvx, operandb=bvy, pickle_size=130, sizeof=56)
+        check(BitvecUnsignedRem, operanda=bvx, operandb=bvy, pickle_size=138, sizeof=56)
+
+        check(BitvecShiftLeft, operanda=bvx, operandb=bvy, pickle_size=136, sizeof=56)
+        check(BitvecShiftRight, operanda=bvx, operandb=bvy, pickle_size=137, sizeof=56)
+        check(BitvecArithmeticShiftLeft, operanda=bvx, operandb=bvy, pickle_size=146, sizeof=56)
+        check(BitvecArithmeticShiftRight, operanda=bvx, operandb=bvy, pickle_size=147, sizeof=56)
+
+        check(BitvecZeroExtend, operand=bvx, size=122, pickle_size=119, sizeof=56)
+        check(BitvecSignExtend, operand=bvx, size=122, pickle_size=119, sizeof=56)
+        check(BitvecExtract, operand=bvx, offset=0, size=8, pickle_size=119, sizeof=64)
+        check(BitvecConcat, operands=(bvx, bvy), pickle_size=133, sizeof=56)
+        check(BitvecITE, condition=x, true_value=bvx, false_value=bvy, pickle_size=161, sizeof=56)
+
+        a = ArrayVariable(index_size=32, value_size=32, length=324, name="name")
+        check(ArrayConstant, index_size=32, value_size=8, value=b"A", pickle_size=136, sizeof=64)
+        check(ArraySlice, array=a, offset=0, size=10, pickle_size=136, sizeof=48)
+        check(ArraySelect, array=a, index=bvx, pickle_size=161, sizeof=56)
+        check(ArrayStore, array=a, index=bvx, value=bvy, pickle_size=188, sizeof=80)
+
+        def all_subclasses(cls):
+            return set((Expression,)).union(
```

Comment:
Nit (as suggested by Pycharm)
```suggestion
            return {Expression}.union(
```